src/js/inverted-index.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * InvertedIndex class
+ * @class
+ */
+class InvertedIndex {
+  /**
+   * class constructor
+   * @constructor
+   */
+  constructor() {
+    this.indexes = {};
+  }
+
+  /**
+   * Get individual words from a string of text.
+   * @function
+   * @param {String} text text to be getWords.
+   * @return {Array} array of string tokens
+   */
+  static getWords(text) {
+    return text.toLowerCase().replace(/[^\w\s]/g, '').split(/\s+/);
+  }
+
+  /**
+   * Create index
+   * @function
+   * @param {string} indextitle
+   * @param {Array} data
+   * @return {Object} index object
+   */
+  createIndex(indextitle, data) {
+    const dictionary = {};
+
+    data.forEach((doc, arrayIndex) => {
+      const docWords = `${doc.title} + ' ' + ${doc.text}`;
+      const words = InvertedIndex.getWords(docWords);
+
+      words.forEach((word) => {
+        if (dictionary[word]) {
+          if (dictionary[word].indexOf(arrayIndex) === -1) {
+            dictionary[word].push(arrayIndex);
+          }
+        } else {
+          dictionary[word] = [arrayIndex];
+        }
+      });
+    });
+
+    this.indexes[indextitle] = dictionary;
+
+    return this.indexes[indextitle];
```

Comment:
Why did you need to return the value of `this.indexes[indextitle]` after assigning it in line 49.

I believe it's already available for use from the `getIndex` method.